components/compliance-service/reporting/relaxting/reports.go

```diff
@@ -284,18 +284,21 @@ func (backend *ES2Backend) GetReports(from int32, size int32, filters map[string
 	reports := make([]*reportingapi.Report, 0)
 	if searchResult.TotalHits() > 0 && searchResult.Hits.TotalHits > 0 {
 		for _, hit := range searchResult.Hits.Hits {
-			item := ESInSpecSummary{}
+			item := ESInSpecReport{}
```

Comment:
Spotted this with Rick on peer reviewing and decided to switch to the right Struct for this Unmarshalling to avoid problems in the future.